### PR TITLE
Create jekyll-scholar))   \   by Sylvester Keil -- extensions for the…

### PR DESCRIPTION
**Pull request by jon1234567**
… blogging scholar
- [**Scholar**](https://github.com/inukshuk/jekyll-scholar)           \
  (gem: [jekyll-scholar](https://rubygems.org/gems/jekyll-scholar))   \
  by Sylvester Keil -- extensions for the blogging scholar

### CODE REVIEW

**Comment by ginfuru** [comment]
very cool 👍

**Comment by ginfuru** [comment]
@jon1234567 Hey looks like you crated an empty directory instead of actually editing the markdown file. If you want to resubmit your pull request I'll go ahead and close this one out.

File: - [**Scholar**](https:/github.com/inukshuk/jekyll-scholar)           \   (gem: [jekyll-scholar](https:/rubygems.org/gems/jekyll-scholar))   \   by Sylvester Keil -- extensions for the blogging scholar [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (jon1234567) [inline - [**Scholar**](https:/github.com/inukshuk/jekyll-scholar)           \   (gem: [jekyll-scholar](https:/rubygems.org/gems/jekyll-scholar))   \   by Sylvester Keil -- extensions for the blogging scholar:1]: 87fa944bd78f75bc42479f6ba3981496f2dfc47b

**Review by jeffreytse — COMMENTED** [review]
Got any problem?

**Pull request closed without merging**